help/develop/getting-started.md

```diff
@@ -3,208 +3,216 @@
+If you already know something about Datagrok (probably thanks to our [YouTube channel]), you can take a fast track:
 
-Let's get it started!
+* Have a look at the [20-minute introduction to a Datagrok application] for a refresher, and then proceed to the
+  [JavaScript programming exercises](exercises.md) that explain how to work with Datagrok using the JavaScript API.
+* Check out the [Daily Sources of Information](#daily-sources-of-information), where you can post questions and get
+  information about the platform updates. Return to Daily Sources of Information from time to time to study Datagrok in
+  depth.
 
-From our detailed range of [recorded user meetings and tutorials](https://www.youtube.com/channel/UCXPHEjOd4gyZ6m6Ji-iOBYg)
-we've curated these which let you start fast with building on Datagrok. Besides getting an overall idea about the platform,
-which will help you a lot all the way further in your work, in these videos you'll be directly introduced to our
-JavaScript API and platform extensions.
+In case you want a clean start, review the following sections one by one, in order:
 
-There's about 5 hours of watching to begin with. It's up to your taste if you watch everything first and then
-proceed to [exercises](#datagrok-tutorials-and-exercises), or prefer combining them on the way.
+1. [Daily Sources of Information](#daily-sources-of-information)
+2. [Datagrok Video Walkthrough](#datagrok-video-walkthrough)
+3. [Frequently Asked Questions](#frequently-asked-questions)
+4. [Datagrok Tutorials](#datagrok-tutorials-and-exercises) or [start using the public version of Datagrok]
+5. [Datagrok Exercises](#datagrok-tutorials-and-exercises) or [start directly](exercises.md)
+6. [Datagrok Architecture](#datagrok-architecture) or [start directly](admin/architecture.md)
+7. [Extending and Customizing Datagrok](#extending-and-customizing-datagrok) or [start extending and customizing Datagrok]
+8. [Building Datagrok Applications](#datagrok-application-development) or [start directly](how-to/build-an-app.md)
 
-1. Start with a [Coffee Company](https://www.youtube.com/watch?v=tVwpRB8fikQ) Feature Demo to get an overall idea
+## Daily Sources of Information
 
-2. Going further, proceed with all videos from [the Platform Overview](video-contents.md/#getting-started)
+Check out the following spaces for more information about Datagrok:
 
-3. Using Datagrok JS API would become your regular routine. Proceed with all videos [the JS API Overview](video-contents.md/#java-script-api)
+1. [Community Forum]. This is where we post updates about the Datagrok platform and ecosystem. We recommend that you
+   subscribe at least to the following three topics on the forum:
+   * [JavaScript API Updates]
+   * [Visualization-related Updates]
+   * [Cheminformatics Updates]
 
-4. Learn about some daily Datagrok tooling:  
+2. [Datagrok Release History] &mdash; this page aggregates the release notes. For every new Datagrok build, the release notes
```

Comment:
@AndSviat is this more common to use an entity like `&mdash` instead of a regular Unicode symbol like `—`? I'm used to typing it from the keyboard directly (using a tool like https://ilyabirman.ru/projects/typography-layout/).

Reply:
According to Microsoft Writing Style, we can use a hyphen `-`, an en dash `&ndash;` or an em dash `&mdash;`, depending on the text we write. So if it's a two-word construction (like `two-word`), we use a hyphen - to separate the words. If we need a minus sign or we want to write a range like this `100500&ndash;100501`, then we need to use an en dash. Finally, for parenthetical information that requires more emphasis, we need to use an em dash (the widest dash).

In Markdown, I use the code representation (&ndash;, &mdash;) for consistency between the browsers.